frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -252,7 +252,7 @@ const TextField = ({
     return (
       <Autocomplete
         freeSolo
-        size="small"
+        fullWidth
```

Comment:
does this impact any of the other uses of autocomplete?

Reply:
no - tested downstreams and it seems the same. do you think we should pass it is a prop and have the default be small, or should it always be `fullWidth`? The reason why I was thinking it should always be fullWidth is that a lot of times people are using it inside some container so they can make it smaller there.